marshal/claim.go

```diff
@@ -316,6 +317,12 @@ func (c *claim) teardown(releasePartition bool) bool {
 // messagePump continuously pulls message from Kafka for this partition and makes them
 // available for consumption.
 func (c *claim) messagePump() {
+	messageBackoff := &backoff.Backoff{
```

Comment:
It might be clearer if you mentioned that the channel has a buffer of size 10k here. It took me a few minutes to realize that this backoff won't cause problems due to the large size of the buffer.